eqllib/actors/GAMAREDON GROUP/G0047-persistence-startup-folder-file-write.toml

```diff
@@ -0,0 +1,22 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "low"
+contributors = ["Daniel Stepanic"]
+created_date = "02/12/2020"
+description = "Adversaries abuse common persistence mechanisms such as placing their malware/implants into a compromised user's startup folder. This detection identifies GAMAREDON GROUP's technique of placing shortcut and VBScript files into this folder."
+id = "5430be26-4019-4bc3-bb04-056019304dc9"
+name = "Startup Folder Persistence with Shortcut/VBScript Files"
+os = ["windows"]
+tactics = ["Persistence"]
+tags = ["Gamaredon Group"]
+techniques = ["T1060"]
+updated_date = "02/12/2020"
+references = ["https://www.elastic.co/blog/playing-defense-against-gamaredon-group"]
+
+[analytic]
+query = '''
+file where subtype.create
+  and (file_path == "*\\Programs\\Startup\\*.lnk" or
```

Comment:
I believe .lnk files are very common in the Startup folder. Ideas to make that higher fidelity? What process typically creates these?

Reply:
Updated the query with some common processes that were used maliciously to insert file in startup folder